### ISSUE TITLE
Mobile layout overlaps on artifact inspection page

### ISSUE THREAD

**Issue by TimvdLippe** [sonatype-nexus-community/search-maven-org#36]
Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**
I am visiting the artifact page on mobile. 
![image](https://user-images.githubusercontent.com/5948271/43954458-e5302984-9c9c-11e8-9c99-0b0b7c078ef7.png)

* **What feature or behavior is this required for?**
To be able to browse the website on mobile.
* **How could we solve this issue? (Not knowing is okay!)**
Update the stylesheet of `div.artifact[_ngcontent-c11] .artifact-title[_ngcontent-c11]` and remove `padding-top: 16px`. Add the following styles:

```css
align-items: center;
display: flex;
min-height: 18px;
```

Also, the text gets wrapped on larger screens: 
![image](https://user-images.githubusercontent.com/5948271/43954710-c6a12044-9c9d-11e8-9a32-c1006beb7383.png)

It seems that the `ellipsis` class should not be on that div. Remove the class and add the following css to the same selector:

```css
max-width: 100%;
white-space: nowrap;
```
And then while we are add it, also add a `padding-right: 5px` to have a little nicer spacing between the title of the artifact and the version dropdown menu
* **Anything else?**
With these instructions, this should be the outcome:
![image](https://user-images.githubusercontent.com/5948271/43954828-39c3f0ce-9c9e-11e8-8237-ced5f6f15023.png)
![image](https://user-images.githubusercontent.com/5948271/43954835-409e2c34-9c9e-11e8-9d5c-bc5940d805f6.png)

**Comment by SirMaster** [comment]
@TimvdLippe thanks for creating this and giving us the spots where to correct. I have updated our code already and am waiting for it to go through our review and build process. 

I did have to make some adjustment to your suggests around spacing as certain pages will have **OSS Index** information between the title / version selection and the pom, as shown below : 
![screen shot 2018-08-10 at 7 13 54 pm](https://user-images.githubusercontent.com/3211896/43994484-67f18f7e-9d52-11e8-9ef6-e92bacccc3a3.png)

Additionally I assured that the `ellipsis` won't haven effect on large screens for the group and artifact id, but will on smaller screens.
